lib/api/httpRoutes.js

```diff
@@ -187,7 +187,7 @@ const routes = [
 
   { verb: 'post', path: '/:index/_create', controller: 'index', action: 'create' },
 
-  { verb: 'post', path: '/:index/:collection/_count', controller: 'document', action: 'count', openapi: {parameters: DocumentCount.post.parameters, responses: DocumentCount.post.responses} },
+  { verb: 'post', path: '/:index/:collection/_count', controller: 'document', action: 'count', openapi: {...DocumentCount} },
```

Comment:
Useless spread operator
```
> o = {a: 2, b: 3}
{ a: 2, b: 3 }
> e = o
{ a: 2, b: 3 }
> e = { ...o }
{ a: 2, b: 3 }
```

```suggestion
  { verb: 'post', path: '/:index/:collection/_count', controller: 'document', action: 'count', openapi: DocumentCount },
```